.vscode/extensions.json

```diff
@@ -0,0 +1,6 @@
+{
+  "recommendations": [
+    "dbaeumer.vscode-eslint",
+    "burkeholland.simple-react-snippets"
+  ]
```

Comment:
looking at if there is a way to have recommended vs required. the first one here is used as the linter so it's necessary, whereas the second helps with autocomplete and isn't used in the settings, so is only recommended.

Reply:
Soo there isn't a way. The closest I found is you can make your own extension which is an extension pack, and every extension within the pack will be downloaded alongside. But, you have to download the original extension containing the packs so there's still nothing really forced. That could be a good option if you have a lot of extensions you work with regularly, but we only have these 2 right now so it's not worth it at this time.

For reference, this is [an example of a extension with an extension pack](https://marketplace.visualstudio.com/items?itemName=waldo.al-extension-pack), and this is [an overview from the creator of the pack](https://www.waldo.be/2021/09/17/how-to-force-vscode-extensions-on-your-teams-pcs/).